installation/upgrade-notes.md

```diff
@@ -1,13 +1,19 @@
 # Upgrade Notes
 
-The following article cover the relevant notes for users upgrading from previous Fluent Bit versions. We aim to cover compatibility changes that you must be aware of.
+The following article cover the relevant notes for users upgrading from previous
+Fluent Bit versions. These aim to cover compatibility changes that you must be aware of.
 
-For more details about changes on each release please refer to the [Official Release Notes](https://fluentbit.io/announcements/).
+For more details about changes on each release, refer to the
+ [Official Release Notes](https://fluentbit.io/announcements/).
 
-Note: release notes will be prepared in advance of a Git tag for a release so an official release should provide both a tag and a release note together to allow users to verify and understand the release contents.
+Release notes will be prepared in advance of a Git tag for a release. An official
+release should provide both a tag and a release note together to allow users to
+verify and understand the release contents.
 
-The tag drives the overall binary release process so release binaries (containers/packages) will appear after a tag and its associated release note.
-This allows users to expect the new release binary to appear and allow/deny/update it as appropriate in their infrastructure.
+The tag drives the binary release process. Release binaries (containers and packages)
+will appear after a tag and its associated release note. This lets users to expect
+the new release binary to appear and allow/deny/update it as appropriate in their
+infrastructure.
```

Comment:
These seem like internal process notes that we shouldn't or don't need to expose to users.

Reply:
I am definitely not the one who made that call :D